index.bs

```diff
@@ -951,7 +951,7 @@ spec:css-syntax-3;
     1.  Assert: |settings| is a [=secure context=].
 
     1.  If |settings|'s [=relevant global object=]'s [=associated Document=] is not [=Document/fully active=],
-        then return [=a promise rejected with=] "{{NotAllowedError}}" {{DOMException}}.
```

Comment:
Actually, we should also make the change for the other algorithms (request, store, prevent silent access, and create), no?

Reply:
oh yes, totally... will make those changes too.